### PR TITLE
Fix for https://github.com/allenai/allennlp/issues/203

### PR DESCRIPTION
**Pull request by murphp15**
Add a wrapper around the learning rate schedulers.

### CODE REVIEW

File: allennlp/training/learning_rate_schedulers.py [hunk 1]
```diff
@@ -22,10 +23,58 @@ class LearningRateScheduler(Registrable):
...
     """
+    def __init__(self, lr_scheduler) -> None:
+        self.lr_scheduler = lr_scheduler
+
     @classmethod
     def from_params(cls, optimizer: torch.optim.Optimizer, params: Params):
         scheduler = params.pop_choice("type", LearningRateScheduler.list_available())
-        return LearningRateScheduler.by_name(scheduler)(optimizer, **params.as_dict())  # type: ignore
+
+        lrs = LearningRateScheduler.by_name(scheduler)(optimizer, **params.as_dict())  # type: ignore
```
Comment (DeNeutoy) [inline allennlp/training/learning_rate_schedulers.py]: Perhaps use `schedulers` for this variable name. In general we prefer un-abbreviated, longer variable names in the codebase.

Comment (murphp15) [reply allennlp/training/learning_rate_schedulers.py]: Cool will change now.

Comment (murphp15) [reply allennlp/training/learning_rate_schedulers.py]: @DeNeutoy I don't have 90% code coverage on the code I added. 
It is in the 80% would you be able to merge it without me meeting the 90% requirement?

Comment (DeNeutoy) [reply allennlp/training/learning_rate_schedulers.py]: Yes no problem - the code coverage is more of a soft requirement and designed to alert us when we for instance write a test which isn't getting run, so we can spot it. This PR is certainly well enough tested.

**Comment by murphp15** [comment]
@DeNeutoy your last change seems to have caused the build to faill?
